webplugin/js/app/km-nav-bar.js

```diff
@@ -34,6 +34,7 @@ function KmNavBar(mckMsgLayout) {
 
     _this.modifyTalkToHumanVisibility = function (isAssignedToHuman) {
         if (isAssignedToHuman) {
+            document.getElementById("km-talk-to-human").disabled = false;
```

Comment:
can this throw undefined error ever if there is no element on UI?

Reply:
it will not throw any error because modifyTalkToHumanVisibility get called when user clicks on the button so if user is able to click on the button that means element is available and it will not throw error.

For safety adding optional chaining.